services/mealSchedule.js

```diff
@@ -0,0 +1,80 @@
+// LOCAL MODULES
+const { getDbConn, } = require('./db/db');
+const { dbAddr, } = require('./db/config');
+
+//ADD SCHEDULED MEAL FOR USER
+const createScheduledMeal = (user_id, recipe_id, day_id) => getDbConn(dbAddr).one(
+    `   
+        INSERT INTO meal_schedule
+        (user_id, 
+        recipe_id, 
+        day_id) 
+        VALUES 
+        ($[user_id], 
+        $[recipe_id], 
+        $[day_id]) RETURNING id;`
+    , { user_id, recipe_id, day_id }
+);
+
+//GET SCHEDULED MEALS FOR SPECIFIC USER ID
+const getScheduledMeals = id => getDbConn(dbAddr).any(
```

Comment:
I would recommend to rename the variable `id` to `user_id` since its easy to confuse this ID to be the `meal_schedule.id`.